pkg/kubelet/kuberuntime/util/util_test.go

```diff
@@ -223,7 +223,7 @@ func TestNamespacesForPod(t *testing.T) {
 		},
 	} {
 		t.Run(desc, func(t *testing.T) {
-			actual, err := NamespacesForPod(test.input, &kubecontainertest.FakeRuntimeHelper{})
+			actual, err := NamespacesForPod(test.input, &kubecontainertest.FakeRuntimeHelper{}, nil)
```

Comment:
This should have user namespace coverage.

Reply:
Yes, but seems beyond the scope of the PR.

cc @giuseppe @rata

Reply:
I'll open a follow-up PR for that, thanks!